Mage.Sets/src/mage/sets/Battlebond.java

```diff
@@ -9,23 +9,27 @@
  * @author TheElk801
  */
 public final class Battlebond extends ExpansionSet {
-
+    
     private static final Battlebond instance = new Battlebond();
-
+    public int i;
+    
     public static Battlebond getInstance() {
         return instance;
     }
-
-    private Battlebond() {
+    
+    
+    public Battlebond() {
         super("Battlebond", "BBD", ExpansionSet.buildDate(2018, 6, 8), SetType.SUPPLEMENTAL);
         this.blockName = "Battlebond";
         this.hasBasicLands = false;
         this.hasBoosters = true;
+        this.PartnerMechanic = true;
```

Comment:
small nitpick, this should be hasPartnerMechanic or something similar

Reply:
Fixed!